static/assets/icons/globe.svg

```diff
@@ -1 +1 @@
"><path d="M7.32.027a8.09 8.09 0 0 0-4.312 1.711c-.352.277-.992.917-1.269 1.267C.835 4.149.289 5.432.062 6.947c-.072.479-.066 1.649.011 2.147.34 2.19 1.403 3.995 3.122 5.299 1.062.806 2.286 1.312 3.711 1.534.523.082 1.665.082 2.188 0 .981-.153 1.874-.442 2.659-.861a8.145 8.145 0 0 0 3.313-3.313c.419-.785.708-1.678.861-2.659.047-.302.056-.48.056-1.094 0-.614-.009-.792-.056-1.094-.222-1.425-.728-2.649-1.534-3.711C13.07 1.452 11.181.356 9 .066A11.252 11.252 0 0 0 7.32.027m1.64 1.376a6.84 6.84 0 0 1 2.463.872l.102.063-.134.086c-.346.222-.502.547-.34.709.095.096.335.156.56.141.345-.022.519-.17.486-.412-.009-.063-.007-.115.004-.115.01 0 .085.059.166.131l.146.131-.19.196a2.613 2.613 0 0 0-.286.346c-.144.227-.188.255-.546.344a.564.564 0 0 0-.213.118c-.095.087-.098.096-.097.272.002.211.063.354.208.481.175.153.514.18.782.061.163-.072.249-.153.409-.387.194-.284.297-.36.49-.36.125 0 .352.122.413.222a.73.73 0 0 1 .071.245c.035.294.117.364.308.265l.08-.041.171.354c.19.394.19.401.014.522-.181.123-.387.047-.747-.274-.427-.381-.585-.413-1.369-.281-.823.139-.926.197-1.27.706-.077.115-.235.31-.349.434-.373.401-.505.667-.505 1.016 0 .525.373 1.09.882 1.334.24.116.548.191.777.191.077 0 .291-.024.474-.054.364-.06.642-.071.749-.03.177.068.264.289.264.671 0 .243.024.373.148.805.071.244.096.397.118.707.034.486.047.599.099.895l.041.233-.154.194a7.59 7.59 0 0 1-.938.944 6.721 6.721 0 0 1-3.617 1.504c-.367.036-1.198.02-1.53-.03-.212-.032-.22-.036-.22-.104 0-.123.106-.485.212-.728.208-.473.501-.841 1.012-1.271.17-.144.406-.363.523-.487.343-.362.509-.753.538-1.264.028-.487-.067-.824-.305-1.081-.202-.217-.393-.349-.928-.638-.523-.283-.664-.373-.878-.561-.25-.218-.418-.337-.588-.416-.143-.065-.193-.074-.493-.079-.279-.006-.365-.018-.526-.075-.321-.113-.423-.096-.817.137-.305.18-.413.211-.589.17-.186-.042-.329-.134-.501-.319-.194-.209-.295-.417-.309-.633-.008-.137 0-.184.047-.26.107-.174.43-.282.836-.283.232 0 .318.038.536.235.098.089.231.181.297.206.11.042.124.043.185.003.113-.075.136-.216.085-.523-.065-.394-.059-.53.03-.662.04-.06.151-.171.246-.247.27-.214.35-.296.405-.419.065-.143.067-.324.008-.534-.059-.212-.038-.316.095-.468.231-.261.483-.401.945-.525.525-.14.697-.223.776-.377.067-.13.028-.262-.147-.497-.286-.383-.578-.618-.769-.618-.126-.001-.264.088-.476.305-.442.455-.967.584-1.228.304-.164-.176-.127-.313.142-.529.206-.165.265-.275.224-.421-.015-.054-.018-.106-.007-.116.01-.009.121-.054.246-.098.183-.066.287-.085.546-.099.366-.021.348-.027.696.268.097.082.233.174.301.203a.635.635 0 0 0 .714-.144c.113-.129.151-.258.137-.466l-.011-.175.295.018c.162.01.421.036.575.059M1.855 7.015c.135.229.309.539.384.69.213.425.395.585.866.762.506.191.729.36.873.666l.089.187.002.653c.002.437.014.693.036.773.086.307.246.494.655.768.285.19.439.344.529.526.094.189.114.284.178.84.047.415.117.853.202 1.273.01.05.007.087-.008.087-.046 0-.365-.137-.659-.284A6.698 6.698 0 0 1 1.508 9.52c-.125-.53-.159-.85-.16-1.493-.001-.586.022-.861.109-1.307.055-.286.06-.294.109-.2.023.044.153.267.289.495"/></svg>
```

Comment:
Let's put this custom SVG under the `static/shared/` directory, so that it's shared for mobile to use.

One good name would be `static/shared/icons/globe.svg`. I'd also be happy with `static/shared/assets/icons/globe.svg`, or any other reasonable name under `static/shared/`.

Background discussion here:
https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/custom.20icons/near/1338192

Reply:
I think this is a good idea -- we should probably do this as a series of prep commits implementing the bit of refactor to move the whole `static/assets/icons` directory to `static/shared/icons`, of course. (I think it shouldn't be hard).

Reply:
Done

Reply:
Looks good, thanks!

Reply:
Added a `README.md` for the new directory documenting briefly how it works, and updated some documentation that still referenced `static/assets/icons`.

We will definitely want to do some cleanup to move `static/assets/icons` (which now just has configuration for the font) somewhere else as a cleanup, but I don't think that's important for this PR.